stix_shifter_modules/qradar/stix_translation/aql_query_translator.py

```diff
@@ -38,28 +38,30 @@ def parse_query(self, data):
             result[START_TIME] = int(float(start_dt_obj)*1000)
         else:
             for label in labels.keys():
-                result[labels[label]] = self.search_for_pattern(data, time_patterns, label)
+                result[labels[label]] = self.__search_for_pattern(data, time_patterns, label)
 
         if not result.get(END_TIME):
             current_time = datetime.now()
             result[END_TIME] = int(float(datetime.strptime(str(current_time), '%Y-%m-%d %H:%M:%S.%f').strftime('%s.%f'))*1000)
 
         return result
 
+    def transform_query(self, data):
```

Comment:
is there any reason `transform_query`  need to overriding?

Reply:
parent's transform_query does antlr parsing for 'stix' language, it is not needed for 'aql' dialect.